adabot_description/urdf/adabot.macros.xacro

```diff
@@ -2,30 +2,6 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="adabot">
 
 
-    <!-- Create the chassis geometry (for visuals and collisions) -->
-    <xacro:property name="chassis_geometry">
-        <origin xyz="0 0 0" rpy="0 0 0" />
-        <geometry>
-            <box size="${ch_length} ${ch_width} ${ch_height}" />
-        </geometry>
-    </xacro:property>
-
-
-
-
-    <!-- Create the wheel geometry (for visuals and collisions) -->
-    <xacro:property name="wheel_geometry">
-        <!-- Cylinders default with their height along the z axis -->
-        <origin xyz="0 0 0" rpy="${PI/2} 0 0" />
-        <geometry>
-            <!-- <cylinder length="${wh_length}" radius="${wh_radius}" /> -->
```

Comment:
I'm a little confused. I was imagining that adabot had a set geometry for collision logic, then if we have `debug_visual:=true` the visual is set to the same as the collision geometry and if it's set to `debug_visual:=false` the visuals are set to the STL meshes, but the collision geometry stays the same.
Where are you saying that this geometry would change to a cylinder instead of a sphere during `debug_visual:=true`? In the visual tag, the collision tag, both?